merlin-server/sql/merlin/tables/plan.sql

```diff
@@ -61,3 +59,16 @@ after update on plan
 for each row
 when (pg_trigger_depth() < 1)
 execute function increment_revision_on_update_plan();
+
+create function raise_duration_is_negative()
+returns trigger
+security definer
+language plpgsql as $$begin
+  raise exception 'invalid negative duration: %', new.duration;
```

Comment:
Would the word "plan" in here somewhere help avoid ambiguity, or do you figure it's always obvious what duration is being referred to?

Reply:
For sure! Just pushed a fix to change this to:
```sql
  raise exception 'invalid plan duration, expected nonnegative duration but found: %', new.duration;
```

Reply:
Resolving but please let me know if you'd like it reworded at all, happy to make a hotfix PR!